### PR TITLE
Prerelease/1.3.5 alpha.1

### PR DESCRIPTION
**Pull request by KarishmaBothara**

### CODE REVIEW

File: scripts/build-release.sh [hunk 1]
```diff
@@ -7,12 +7,14 @@ printf '\n\nChecking TS definitions..\n\n'
...
-    printf "\n\nBuilding @cennznet/$d...\n\n"
+for d in $(pwd)/packages/* ; do
+    printf "\n\nBuilding $d...\n\n"
     npx babel -f babel.config.js --ignore='**/*.d.ts' --extensions='.ts' --out-file-extension='.js' --copy-files -d "$d/build" "$d/src"
     # copy in essential files for publishing to npm
     cp LICENSE $d/build/LICENSE
-    cp $d/{README.md,package.json} $d/build
+    ls $d
+    cp $d/{README.md} $d/build
+    cp $d/{package.json} $d/build
```
Comment (holygits) [inline scripts/build-release.sh:17]: lets take no changes 😅 
```suggestion
    cp $d/README.md $d/build/README.md
    cp $d/package.json $d/build/package.json
```

**Comment by holygits** [comment]
needs rebase

File: scripts/build-release.sh (hunk 1 above)
Comment (KarishmaBothara) [reply scripts/build-release.sh:17]: ok :) 
Trying to rebase.. having some trouble..

Comment (holygits) [reply scripts/build-release.sh:17]: something like `git fetch origin; git rebase origin/develop` should do it

**Review by holygits — APPROVED** [review]

**Pull request merged**